File: frontend/src/app/views/profile/request.ts
export class Request {

  public ID: number;
  public place: number;
  public clas: number;
  public approved: boolean;
  public price: number;
  public IDROOM: number;

  constructor(ID: number, place: number, clas: number, approved: boolean, price: number, IDROOM: number) {
    this.ID = ID;
    this.place = place;
    this.clas = clas;
    this.approved = approved;
    this.price = price;
    this.IDROOM = IDROOM;
  }
}

File: servlet/src/main/java/com/example/entities/Request.java
package com.example.entities;

import com.fasterxml.jackson.annotation.JsonPropertyOrder;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@JsonPropertyOrder({"ID", "place", "clas", "approved","price", "IDROOM"})
public class Request {
    private int ID;
    private int place;
    private int clas;
    private boolean approved;
    private float price;
    private int IDROOM;
}
